app/src/main/res/layout/layout_nav_header.xml

```diff
@@ -13,26 +13,43 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     android:theme="@style/ThemeOverlay.AppCompat.Dark">
 
-    <ImageView
-        android:id="@+id/imageView"
-        android:layout_height="wrap_content"
-        android:layout_width="wrap_content"
-        android:paddingTop="@dimen/nav_header_vertical_spacing"
-        app:srcCompat="@android:drawable/sym_def_app_icon"/>
-
-    <TextView
-        android:layout_height="wrap_content"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:paddingTop="@dimen/nav_header_vertical_spacing"
-        android:text="Rajan Maurya"
-        android:textColor="@color/white"
-        android:textAppearance="@style/TextAppearance.AppCompat.Body1"/>
+        android:layout_height="match_parent"
+        android:gravity="center"
+        >
+
+        <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:src="@drawable/mifos_transparent"
+            android:layout_marginLeft="100sp"
+            android:layout_marginBottom="50sp"
+            />
+        <ImageView
+            android:id="@+id/imageView"
+            android:layout_height="70sp"
```

Comment:
Don't use these hard-coded height and width, instead use wrap_content or match_parent.